src/components/tables/cells/ChipList.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Chip, styled, TableCell } from '@mui/material';
```

Comment:
New call that can display lists of strings in chips. I think eventually this might expand and they become elements that the user can take action on, select some, select all, deselect all, etc. But right now they just display the text.